smoke/LeafDevice/details/Details.cs

```diff
@@ -25,6 +25,13 @@ namespace LeafDeviceTest
     using Message = Microsoft.Azure.Devices.Client.Message;
     using ServiceClientTransportType = Microsoft.Azure.Devices.TransportType;
 
+   public enum DeviceProtocol {
```

Comment:
should we use Microsoft.Azure.Devices.Client.TransportType?

Reply:
That has more options than a user would want to set for our tests for example Http so it becomes a bit clunky.